tracer/build/_build/Build.Profiler.Steps.cs

```diff
@@ -209,7 +209,7 @@ partial class Build
                     // .EnableNoRestore()
                     .EnableNoDependencies()
                     .SetConfiguration(BuildConfiguration)
-                    .SetTargetPlatform(MSBuildTargetPlatform.x64)
+                    // .SetTargetPlatform(MSBuildTargetPlatform.x64)
```

Comment:
Does this need to be re-enabled?

Reply:
No... this is one of those random changes I had to make that I totally don't understand 😬

Reply:
Sounds good!